Add AWS download instructions for Kaptain in Main (#4099)
mesosphere/dcos-docs-site: pages/dkp/kaptain/1.3.0/download/index.md

<!-- markdownlint-disable MD034 -->

> Kaptain downloads are only available to licensed customers. To obtain an evaluation license for Kaptain, please contact <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.

Reviewer: ```suggestion
Kaptain downloads are only available to licensed customers. To get an evaluation license for Kaptain, contact <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

If you already have a license, downloads are available in the [Support Portal][support-portal].

## Download from the AWS Marketplace

Follow the instructions on AWS console to download the container image.

After downloading the image, run the following command to copy the binaries onto your host.

```sh
docker run -it --rm -v $(pwd):/kaptain <container_image_name>
```

> You will then see the following output:

Reviewer: ```suggestion
You then see the following output:
```

```sh
Kaptain install package placed in the local directory. See install instructions at: https://docs.d2iq.com/dkp/kaptain/
```

> You will now see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.

Reviewer: ```suggestion
You see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```
